packages/frontend/src/components/buy/BuyNear.js

```diff
@@ -174,7 +185,7 @@ export function BuyNear({ match, location, history }) {
             (e) => console.warn('Error checking Moonpay', e)
         );
     };
-
+    nearpay.isAvailable = false;
```

Comment:
Are we not ready to release this? Or is this leftover debugging code?

Reply:
@andy-haynes This is not a leftover. The flag is needed because another approval from NearPay's side is required before going live on mainnet. Maybe it's a good idea to use a feature-flag for this case